src/Umbraco.Web/PropertyEditors/ValueConverters/MediaPickerWithCropsValueConverter.cs

```diff
@@ -0,0 +1,152 @@
+﻿using Newtonsoft.Json;
+using System;
+using System.Collections;
+using System.Collections.Generic;
+using System.Runtime.Serialization;
+using Umbraco.Core.Models.PublishedContent;
+using Umbraco.Core.PropertyEditors;
+using Umbraco.Core.PropertyEditors.ValueConverters;
+using Umbraco.Web.PublishedCache;
+
+namespace Umbraco.Web.PropertyEditors.ValueConverters
+{
+    [DefaultPropertyValueConverter]
+    public class MediaPickerWithCropsValueConverter : PropertyValueConverterBase
+    {
+        // JSON is an array of the following JSON
+        /*
+        "key": "d1e9c080-5aef-48f6-9b37-47e84de7ad8b",
+        "mediaKey": "80e2507b-c5f9-4c8a-ab14-5dae717a49b3",
+        "crops": [
+            {
+                "alias": "square",
+                "label": "Square",
+                "width": 200,
+                "height": 200,
+                "coordinates": {
+                    "x1": 0.4489663034789123,
+                    "y1": 0.7192332259400422,
+                    "x2": 0.39324276949939146,
+                    "y2": 0
+                }
+            },
+            ...
+        ]
+        "focalPoint": {
+            "left": 0.846,
+            "top": 0.7964285714285714
+        }
+        */
+
+        private IPublishedSnapshotAccessor _publishedSnapshotAccessor;
+
+        public MediaPickerWithCropsValueConverter(IPublishedSnapshotAccessor publishedSnapshotAccessor)
+        {
+            _publishedSnapshotAccessor = publishedSnapshotAccessor ?? throw new ArgumentNullException(nameof(publishedSnapshotAccessor));
+        }
+
+        public override PropertyCacheLevel GetPropertyCacheLevel(IPublishedPropertyType propertyType) => PropertyCacheLevel.Snapshot;
+
+        /// <summary>
+        /// Enusre this property value convertor is for the New Media Picker with Crops aka MediaPicker 3
+        /// </summary>
+        public override bool IsConverter(IPublishedPropertyType propertyType) => propertyType.EditorAlias.Equals(Core.Constants.PropertyEditors.Aliases.MediaPicker3);
+
+        /// <summary>
+        /// Check if the raw JSON value is not an empty array
+        /// </summary>
+        public override bool? IsValue(object value, PropertyValueLevel level) => value?.ToString() != "[]";
```

Comment:
This seemed like a bit of an odd way to check if the json is not an empty array, but then again I do suppose it works, and is better than trying to cast/deserialize into a JObject or something, so maybe it's fine, it just sticked out to me.

Reply:
Yeh I think it was a copy/paste job from `MultiUrlPickerValueConvertor` if you think we should need more battle tested code to check if the value is not an empty JSON array let me know.

Reply:
Ahh yeah I see that now. Well I suppose since that has been a thing for a couple of years now, it should be battle tested, so I think we can just send it as is.